Automattic/wp-calypso
# Dotcom: New Sites Dashboard v2 using the ItemsDataViews from A4A

Related to #

## Proposed Changes

Base code to establish the foundation of the new Sites Dashboard V2, using the ItemsDataViews and the ItemPreviewPane components.

**Context**: p1713887511207539-slack-C06TF5T7MV5

The new Sites Dashboard V2 is behind a feature flag. Use this query URL: `?flags=layout/dotcom-nav-redesign-v2`

## Testing Instructions

There are no specific instructions for testing this. This is the code base, the foundation to continue working on top of it. Any issue, known or not, will be addressed in follow-up PRs.

More context about the current status and known issues: p1713887511207539-slack-C06TF5T7MV5

- Check the code to familiarize yourself with it.
- Test that the current `/sites` page works as before, without any regressions.
- The new Sites Dashboard V2 is behind this feature flag: `layout/dotcom-nav-redesign-v2` so you have to add this query param to your URL `?flags=layout/dotcom-nav-redesign-v2`
- In `/sites` you have see sites
- If you click on the chevron icon of a site, it will open the Preview Pane of the Site.
- With the Preview Pane opened, you can click on each tab and see the content.
- The first tab is the Hosting Overview tab with the content which is rendered by the `HostingOverview` component

## Pre-merge Checklist
